Bindings/Java/Matlab/Utilities/osimC3D.m

```diff
@@ -81,11 +120,86 @@ function rotateData(obj,axis,value)
             % rotate the tables
             obj.rotateTable(obj.markers, axis, value);
             obj.rotateTable(obj.forces, axis, value);
-            
             disp('Marker and Force tables have been rotated')
         end
+        function writeTRC(obj,varargin)
+            % Write marker data to trc file.
+            % osimC3d.writeTRC()                       Write to current dir using c3d name
+            % osimC3d.writeTRC('Walking.trc')          Write to current dir with input name
+            % osimC3d.writeTRC('C:\data\Walking.trc')  Write to input location from full input path
+
+            % Compute an output path to use for writing to file
+            outputPath = generateOutputPath(obj,varargin,'.trc');
+
+
+            import org.opensim.modeling.*
+            % Write to file
+            TRCFileAdapter().write( obj.getTable_markers, outputPath)
```

Comment:
You don't need to make a clone of the markers table here. You could use `obj.markers` instead of `obj.getTable_markers`. Also, it is good practice to use parentheses when invoking methods, even if there are no arguments (`obj.getTable_markers()`).